WBooks/TableBooks/TableBookController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  TableBookController.swift
+//  WBooks
+//
+//  Created by Juan Silva on 04/06/2021.
+//
+
+import Foundation
+import UIKit
+
+final class TableBookController: UITableViewController {
+    private lazy var tableBookView = TableBookView()
+    
+    struct Book {
+        var title: String
+        var autor: String
+        var image: String
+    }
+    
+    override func loadView() {
+        view = tableBookView
+    }
+    
+    var books: [Book] = [Book(title: "A Little Bird Told Me", autor: "Timothy Cross", image: "img_book1.png"), Book(title: "When the Doves Disappeared", autor: "Sofi Oksanen", image: "img_book2.png"), Book(title: "The Best Book in the World", autor: "Peter Sjernstrom", image: "img_book3.png"), Book(title: "Be Creative", autor: "Tony Alcazar", image: "img_book4.png"), Book(title: "Redesign the Web", autor: "Liliana Castilla", image: "img_book5.png")]
+
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        tableBookView.booksTable.delegate = self
+        tableBookView.booksTable.dataSource = self
+        let nib = UINib(nibName: CellBookView.identifier, bundle: nil)
+        tableBookView.booksTable.register(nib, forCellReuseIdentifier: CellBookView.identifier)
+
+
+    }
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+//        return books.count
+        return 1
+    }
+    
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return books.count
+    }
+
+    override func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        return 10
+    }
```

Comment:
I suspect this has to do with the spaces between cells. I think this can be achieved by adding some padding in the xib file, but I'll take a look at it in a sec.